sqlglot/parser.py

```diff
@@ -1157,13 +1157,16 @@ def _parse_create(self) -> t.Optional[exp.Expression]:
         begin = None
         clone = None
 
+        def extend_props(temp_props):
```

Comment:
Let's add some type hints here.

Reply:
Added

Reply:
Thanks

Reply:
FYI the type signature in the snippet I pasted is not consistent with the types in current main branch because I've fixed several type hints. You can use this to fix the error:

```python
def _parse_properties(self, before: t.Optional[bool] = None) -> t.Optional[exp.Properties]:
```